Sunflower/app/src/main/java/com/jin/sunflower/core/data/local/InMemoryLocalGardenDataSource.kt

```diff
@@ -0,0 +1,15 @@
+package com.jin.sunflower.core.data.local
+
+import com.jin.sunflower.core.model.Plant
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+class InMemoryLocalGardenDataSource {
+
+    private val _myGardenList = MutableStateFlow(emptyList<Plant>())
+    val myGardenList: StateFlow<List<Plant>> = _myGardenList
+
+    fun addedPlantToGarden(plant: Plant) {
```

Comment:
```suggestion
    fun addPlantToGarden(plant: Plant) {
```
과거 분사는 예전에 있었던 특정 이벤트의 의미를 갖습니다.
명령형은 1인칭 동사를 사용하세요.